databases/week2/part1.sql

```diff
@@ -0,0 +1,18 @@
+USE hyf_class2023;
+-- add a task
+insert into task (title, description, created, updated, due_date, status_id, user_id) values ('DB week 2 home work', 'Insert a new task', '2017-10-20 01:41:53', '2017-10-04 07:19:56', '2017-12-23 10:13:42', 2, NULL);
```

Comment:
I don't think you are able to insert this data into the task table as `user_id` cannot be `NULL`. Please check the query from week 1 we use to create a user table.